city_local/berlin/_berlin_events/1.md

```diff
@@ -0,0 +1,43 @@
+---
+title:  "First Event 🎉"
+date:   2019-06-20 15:26:13 +0100
+---
+
+## Tech Workers Coalition Berlin
+Tech Workers have a lot of potential power to change society. Anyone who identifies as a _tech worker_ and is interested in learning how to build collective power or get involved in a campaign is welcome. Tech Workers are currently organising against gender pay gaps, workplace harassment, surveillance technologies, amongst other things. This document is an agenda item for the first meeting of Tech Workers Coalition Berlin.
+
+A panel about _Tech Workers Movement_ was held earlier this month 10 June 2019 at Aquarium Am Südblock, featuring Logic Magazine co-founders [Ben Tarnoff](https://twitter.com/bentarnoff) and [Moira Weigel](moiragweigel).  Ben, who is also an organiser of the Boston chapter, will be present during the meeting. You can click to watch the [video](https://www.facebook.com/halbzehn.fm/videos/423601581824448/) or listen for [audio](https://soundcloud.com/rosaluxstiftung/the-new-tech-worker-movement).
+
+## 20 June 2019
+This will be the first of many future meetups. A second meetup is scheduled for the 27th of June, so do not worry if you missed the first one!
+
+## Venue
+
+[Bilgisaray, Oranienstrasse 45, Berlin](https://www.google.com/maps/place/Bilgisaray/@52.499971,13.4204474,17z/data=!3m1!4b1!4m5!3m4!1s0x47a84e34f7d3f0db:0x4a368a3631962abc!8m2!3d52.499971!4d13.4226362)
+
+Bilgisaray is a smoke-free environment. Unfortunately, it is not wheelchair accessible. We will try to ensure that future venue spaces are accessible.
+
+## Agenda
+
+* 18:45 - Everyone arrives, write name tags
+* 19:00 - People briefly introduce themselves
+* 19:15 - a short presentation by [Yonatan Miller](https://twitter.com/@shushugah) on what Tech Workers Coalition around the world have accomplished
+* 19:30 - Interactive breakout session to figure out what tech/social issues are personally important to you
+* 19:45 - Group report back and discussion
+* 20:00 - Second breakout session to figure out what new/existing campaigns we can get involved in
+* 20:30 - Thirty minutes to wrap up and figure out the next steps for meeting on 27 June and any action items
+* 21:00 - Go home with some resources, contacts and feeling inspired! 💪🏼💪🏽💪🏾💪🏿
+
+## Code of Conduct
+The meeting is covered under the [Berlin Code of Conduct](https://berlincodeofconduct.org/). Additionally, steps will be taken to ensure the privacy and safety of members attending. Sharing the identity of members and or taking photos are strictly forbidden unless express consent is given.
+
+
+## Resources
+
+Twitter: [@TechWorkersBER](https://twitter.com/TechWorkersBER)
+
+Tech Workers Coalition [Facebook](https://www.facebook.com/TechWorkersCoalition)
+
+## Contact info
+
+Listserv: To subscribe send an email to [techworkersberlin-subscribe@lists.riseup.net](mailto:techworkersberlin-subscribe@lists.riseup.net)
```

Comment:
Is there a way we can get this tied to a @techworkerco.org email domain? I manage that and could help on the domain side if there's a way to customize with riseup.net

Reply:
Custom domains, no. Email aliasing/forwarding yes. There are 3 emails that need to be supported. 
```
techworkersberlin-subscribe@lists.riseup.net   -> berlin.subscribe@techworkersco.org
techworkersberlin-unsubscribe@lists.riseup.net -> berlin@techworkersco.org
techworkersberlin@lists.riseup.net             -> berlin.unsubscribe@techworkersco.org
```

Reply:
Also, if we could add checkboxes for different cities people want to subscribe to in the /subscribe page, we could have our backend automatically send a subscription request, reducing hurdle to join locals' listservs.